src/tests/utils.cairo

```diff
@@ -24,7 +24,7 @@ fn pop_log<T, +Drop<T>, +starknet::Event<T>>(address: ContractAddress) -> Option
     let (mut keys, mut data) = testing::pop_log_raw(address)?;
 
     // Remove the event ID from the keys
-    keys.pop_front();
+    keys.pop_front()?;
```

Comment:
```suggestion
    let _ = keys.pop_front();
```

What do you think about using `let _ =` instead? We don't want to return None as event if there are no keys.

Reply:
That's a good point sorry I didn't analyzed well the context.

Reply:
No reason to be sorry, thanks again for taking the time to contribute.